src/routers/datatypeRouters/edges/variants_variants.ts

```diff
@@ -41,6 +49,43 @@ const variantLDQueryFormat = z.object({
   verbose: z.enum(['true', 'false']).default('false')
 })
 
+export async function findVariantLDSummary(input: paramsFormatType): Promise<any[]> {
+  const originalPage = input.page as number
+
+  let limit = QUERY_LIMIT
+  if (input.limit !== undefined) {
+    limit = (input.limit as number <= MAX_PAGE_SIZE) ? input.limit as number : MAX_PAGE_SIZE
+    delete input.limit
+  }
+
+  input.page = 0
+  const variant = (await findVariants(input))
+
+  if (variant.length === 0) {
+    throw new TRPCError({
+      code: 'NOT_FOUND',
+      message: 'Variant not found.'
+    })
+  }
+
+  const id = `variants/${variant[0]._id}`
+
+  const query = `
+  FOR record IN ${ldSchemaObj.db_collection_name}
+    FILTER (record._from == '${id}' OR record._to == '${id}')
+    LET otherRecordKey = PARSE_IDENTIFIER(record._from == '${id}' ? record._to : record._from).key
```

Comment:
There is a ")" in `record._from).key`, but I don't see the corresponding "(".

Reply:
the opening parenthesis is right after PARSE_IDENTIFIER

Reply:
oh, I miss it!